src/main/java/com/github/awwkoala/selenium/page/Page.java

```diff
@@ -0,0 +1,15 @@
+package com.github.awwkoala.selenium.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public abstract class Page {
```

Comment:
CommonPage BasePage BasicPage I'm familiar with those names for the abstraction ;) feel free to leave as you like